Classes/commands.py

```diff
@@ -1029,3 +1032,78 @@ async def team_json(self, ctx):
 
         # Send the JSON file
         await send_long(ctx.channel, json.dumps(json_out), code_block=True)
+
+    @checks.is_admin_bot_channel()
+    @checks.is_white_shirt()
+    @commands.command()
+    # Reimplementation of old ?count_officers command
```

Comment:
The command is missing a doc string, go ahead and look at the other commands to see how that is done, it is what creates information for the ?help command.

Reply:
Added doc string in commit